src/modules/combinedFacets/CombinedFacetsContainer.js

```diff
@@ -70,7 +219,8 @@ class CombinedFacetsContainer extends React.PureComponent {
 
         const combinedFacets = [
             ...formattedTextFacets,
-            ...formattedRangeFacets
+            ...formattedRangeFacets,
+            ...multilevelFacets
```

Comment:
how are we utilising the new core sdk methods here? [ref](https://github.com/manu-unbxd/search-JS-core/blob/eda4de1bb72bb84f79b9c17bcabf1fc9f0950b49/src/modules/facets/getAllFacets.js)

Reply:
discussed this issue offline.